tox.ini

```
@@ -937,52 +943,79 @@ commands =
+    tox -e dev.k8s.registry
+    sleep 3
+    bash -c 'URL=http://k3d-registry.localhost:12345/v2/_catalog; curl -X GET $URL'
     bash -c 'k3d cluster create syft-dev -p "8080:80@loadbalancer" --registry-use k3d-registry.localhost:12345; \
-        k3d cluster list'
+        kubectl create namespace syft;'
+    tox -e dev.k8s.info
 
 [testenv:dev.k8s.deploy]
+description = Deploy to a local Kubernetes cluster with Devspace
 changedir = {toxinidir}/packages/grid
 passenv=HOME, USER
 allowlist_externals =
     bash
 commands =
-    bash -c '\
-        devspace use context k3d-syft-dev; \
-        devspace use namespace syft; \
-        devspace deploy --var CONTAINER_REGISTRY=k3d-registry.localhost:12345 --config=devspace.yaml'
+    bash -c 'devspace deploy -b --kube-context k3d-syft-dev --namespace syft --var CONTAINER_REGISTRY=k3d-registry.localhost:12345'
+
+[testenv:dev.k8s.info]
+description = Gather info about the localKubernetes cluster
+passenv=HOME, USER
+ignore_errors = True
+allowlist_externals =
+    k3d
+    kubectl
+commands =
+    kubectl config view
+    k3d cluster list
+    kubectl cluster-info
+    kubectl config current-context
+    kubectl get namespaces
 
-[testenv:dev.k8s.cleanup]
+[testenv:dev.k8s.destroy]
+description = Destroy local Kubernetes cluster
 changedir = {toxinidir}/packages/grid
+passenv=HOME, USER
 allowlist_externals =
     bash
 commands =
     bash -c '\
-        devspace use context k3d-syft-dev; \
-        devspace use namespace syft; \
-        devspace purge; \
+        devspace purge --kube-context k3d-syft-dev --namespace syft; \
         rm -rf .devspace; echo ""; \
         k3d cluster delete syft-dev; echo ""; \
-        k3d registry delete registry.localhost; echo "";\
         kubectl config view'
+
+[testenv:dev.k8s.destroyall]
+description = Destroy both local Kubernetes cluster and registry
+changedir = {toxinidir}
+passenv=HOME, USER
+ignore_errors=True
```

Q: Why do we ignore error at this place?